Source/Orts.Simulation/Simulation/RollingStocks/MSTSLocomotive.cs

```diff
@@ -2263,7 +2263,7 @@ public override void Update(float elapsedClockSeconds)
             UpdateHornAndBell(elapsedClockSeconds);
 
             UpdateSoundVariables(elapsedClockSeconds);
-            PrevMotiveForceN = MotiveForceN;
+            PrevTractiveForceN = MotiveForceN;
```

Comment:
Is this supposed to be `PrevTractiveForceN = TractiveForceN`?

Reply:
Sure! I don't know how I missed it